src/components/MediaSettings/MediaSettings.vue

```diff
@@ -572,6 +556,11 @@ export default {
 				this.isInTalkDashboard = true
 			}
 
+			if (page === 'device-settings') {
```

Comment:
It's not really different from existing `device-check`. Let's just combine them

`<template v-if="isInTalkDashboard">` block should be removed; it's not a part of checking how camera is working on dashboard, let's leave it only in Talk settings

Reply:
Make sure whatever will be left after combining is properly reset in `closeModal`. Otherwise you can't start a call =O
![image](https://github.com/user-attachments/assets/9bd1437e-9b16-4709-9c62-04ba8b2c4681)